content/homepage/resources.md

```diff
@@ -4,8 +4,15 @@ weight: 2
 button: 'Github Project'
 buttonLink: 'https://github.com/camharris/volatility-oracle'
 ---
+
```

Comment:
Could you please remove these lines 07-12, after the merge we only want the images/links below.

Reply:
Done !